chainer/training/triggers/interval.py

```diff
@@ -40,9 +42,16 @@ def __call__(self, trainer):
         """
         updater = trainer.updater
         if self.unit == 'epoch':
-            prev = self.count
-            self.count = updater.epoch_detail // self.period
-            return prev != self.count
+            epoch_detail = updater.epoch_detail
+            previous_epoch_detail = updater.previous_epoch_detail
```

Comment:
> What is node(s)? Could you share me some examples?

Sorry, this was an undefined term. What I have in mind is that we have a abstract computational graph that consists of **nodes** and **operators** and we regard a computational graph (as a module of Chainer) made by forward propagation as a realization of the part of the abstract graph. Nodes in the abstract computational graph correspond to `chainer.Variable`s and operators to `chainer.Function`s. 

For example, suppose we change operations to be applied to the intermediate variable depending on its value, abstract computational graph will have a branching and each forward propagation will realize either of the branches.